framework/view/ssr/jsx.gotext

```diff
@@ -1,4 +1,4 @@
-import { createView } from "./bud/view/_jsx.ts"
+import { createView } from "./bud/view/_jsx.ts.js"
```

Comment:
Not having `.js` prefix was causing some issues in ecosystem tooling that uses the extension for certain behavior.